lms/static/sass/_build-lms.scss

```diff
@@ -8,6 +8,7 @@
 // base - assets
 @import 'base/font_face';
 @import 'base/extends';
+@import 'base/utilities';
```

Comment:
NIT: I'd move this so its after the ``base/animations`` import (just in case in the future any utilities leverage those animations).